work_queue/src/work_queue_worker.c

```diff
@@ -2498,27 +2501,76 @@ struct list *parse_manager_addresses(const char *specs, int default_port) {
 	return(managers);
 }
 
-void start_coprocess()
-{
+void start_coprocess() {
+
+	if (pipe(coprocess_in) || pipe(coprocess_out)) { // create pipes to communicate with the coprocess
+		debug(D_WQ, "couldn't create coprocess pipes: %s\n", strerror(errno));
```

Comment:
I think we want this one to be fatal(), rather than debug().

Reply:
Change is pushed.